extensions/amp-analytics/0.1/vendors/parsely.js

```diff
@@ -17,20 +17,26 @@
 import {jsonLiteral} from '../../../../src/json';
 
 const PARSELY_CONFIG = jsonLiteral({
+  'vars': {
+    'data': '{}',
+  },
   'requests': {
     'host': 'https://srv.pixel.parsely.com',
     'basePrefix':
       '${host}/plogger/?' +
       'rand=${timestamp}&' +
+      'plid=${pageViewId}&' +
```

Comment:
Note that by using the built-in `${pageViewId}` variable substitution here, AMP pageviews will only send 4-digit `plid` values (such as `8726`), as opposed to the 8-digit values sent by the Parse.ly tracker (such as `54412420`). Other than the integer length, both systems use the same approach:

- Parse.ly tracker: https://github.com/Parsely/parsely-js/blob/352ea7d554d821ddc75a079022b261f871ab232a/js-api/src/tracker/pixel.js#L142-L147
- AMP: https://github.com/ampproject/amphtml/blob/45d40823de8476b3caaa69a04f96c8d18d1f6d7a/src/service/document-info-impl.js#L131-L133

After a GitHub search across Parsely projects, I think this will be OK - we do need a value that can be cast to an `int`, due to our DPL schema: https://github.com/Parsely/parsely_raw_data/blob/303ae23349a888c1c821e4b9779eca245e42404b/parsely_raw_data/schema.py#L67